draft-ietf-openpgp-pqc.md

```diff
@@ -1707,6 +1708,161 @@ Andreas Huelsing (TU Eindhoven)
 
 --- back
 
+# Test Vectors
+
+To help implementing this specification a set of non-normative examples follow here.
+
+## Sample v6 PQC Subkey Artifacts
+
+Here is a Private Key consisting of:
+
+- A v6 Ed25519 Private-Key packet
+- A User ID packet
```

Comment:
v6 certificates need a direct key self-sig on the primary key (see [crypto-refresh §10.1.1](https://www.ietf.org/archive/id/draft-ietf-openpgp-crypto-refresh-13.html#name-openpgp-v6-certificate-stru))

Reply:
Right, missed this in the description. I will push now a commit to fix